mav_trajectory_generation/include/mav_trajectory_generation/polynomial.h

```diff
@@ -235,6 +236,12 @@ class Polynomial {
     return data_size + kernel_size - 1;
   }
 
+  // Scales the polynomial in time with a scaling factor.
+  // To stretch out by a factor of 10, pass scaling_factor (b) = 1/10. To shrink
```

Comment:
Why this way round, saying scaling by 10 stretches by a factor of 10 seems more intuitive to me